netket/nn/linear.py

```diff
@@ -314,6 +318,95 @@ def __call__(self, inputs: Array) -> Array:
         return y
 
 
+class DenseEquivariant(Module):
+    """An equivariant layer that takes a feature map with embeddings corresponding to various
+    poses of a symmetry group, shape = [n_symm*in_features] and outputs a feature map of identical
+    shape [n_symm*out_features]. This transformation respects the group structure such that
```

Comment:
identical_shape but `[n_symm*in_features]` and `[n_symm*out_features]` are clearly different?
What do you mean?

Reply:
Good catch